### PR TITLE
Event Message Center

### PR DESCRIPTION
**Pull request by opolancoh-gl**
There is a component called 'EventMessageCenterUpdateBox.vue' that is not used at this moment. It could help to improve the 'EventMessageCenterUpdate.vue', but this can be implemented in other development iteration.

### CODE REVIEW

File: src/components/events/details/EventDetails.vue [hunk 1]
```diff
@@ -8,13 +8,13 @@ import { mapState, mapActions } from 'vuex';
 export default {
   name: 'EventDetails',
   props: {
-    id: { type: [String, Number] },
+    id: { type: String, required: true },
   },
   created() {
-    console.log('EventDetails created');
+    console.log(`EventDetails created. Id '${this.id}'`);
```
Comment (ogranadagl) [inline src/components/events/details/EventDetails.vue]: remove console.logs please

File: src/components/events/details/EventDetails.vue [hunk 2]
```diff
@@ -8,13 +8,13 @@ import { mapState, mapActions } from 'vuex';
...
   },
   mounted() {
-    console.log('EventDetails mounted');
+    console.log(`EventDetails mounted. Id '${this.id}'`);
```
Comment (ogranadagl) [inline src/components/events/details/EventDetails.vue]: remove console.log please.

**Review by ogranadagl — APPROVED** [review]

File: src/components/events/details/EventDetails.vue (hunk 1 above)
Comment (opolancoh-gl) [reply src/components/events/details/EventDetails.vue]: Done!

File: src/components/events/details/EventDetails.vue (hunk 2 above)
Comment (opolancoh-gl) [reply src/components/events/details/EventDetails.vue]: Done!

**Pull request merged**